.github/workflows/e2e-test.yml

```diff
@@ -44,6 +44,12 @@ jobs:
         exclude:
           - test_groups: 'blocks'
             test_branches: 'merchant'
+          - woocommerce: '5.2.0'
+            test_groups: 'blocks'
+            test_branches: 'shopper'
+          - woocommerce: '6.1.0'
+            test_groups: 'blocks'
+            test_branches: 'shopper'
```

Comment:
@shendy-a8c This approach has some problems. This does skip the specified tests. However the setup script still installs and activates WC Blocks on other tests with WC `5.20` & `6.1.0` which will cause problems.

That why I added the conditional workflow script, which skips installation of WC blocks for specified WC versions.

Reply:
Whoops. My bad. I think I made these changes 😂 to fix the pipeline. 

The idea is to hide the tests are not running from the list.